tiledb/tests/test_basic_import.py

```diff
@@ -2,12 +2,32 @@
 import subprocess
 import sys
 
+import importlib_metadata
+from packaging.version import Version
+
 
 def test_dont_import_pandas() -> None:
     """Verifies that when we import TileDB, we don't import Pandas eagerly."""
+
+    # If tiledb.cloud < 0.10.21 is installed, we should remove it from the module list
+    # before running the test, because it eagerly imported pandas before that version.
```

Comment:
maybe phrase this as "prevent it from being imported" rather than "remove it from the module list"